### PR TITLE
Replace deprecated types and consolidate development testnets

### PR DESCRIPTION
**Pull request by simonsso**
Replace deprecated types FungibleAdapter and GenesisSpec::builder
Consolidate local dev testnets to once config

### CODE REVIEW

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -196,27 +164,21 @@ pub fn development_config(id: ParaId) -> ChainSpec {
...
-		// ID
-		"para_eden_dev",
-		ChainType::Development,
-		move || development_config_genesis(id),
-		Vec::new(),
-		None,
-		None,
-		None,
-		None,
+	ChainSpec::builder(
```
Comment (aliXsed) [inline node/src/chain_spec.rs:167]:
```suggestion
	#[allow(deprecated)]
	ChainSpec::builder(
```

File: node/src/chain_spec.rs [hunk 2]
```diff
@@ -241,30 +203,20 @@ pub fn local_testnet_config(id: ParaId) -> ChainSpec {
...
-		// Telemetry
-		None,
-		// Protocol ID
-		Some("eden-local"),
-		// Fork ID
-		None,
-		// Properties
-		Some(properties),
-		// Extensions
+	ChainSpec::builder(
```
Comment (aliXsed) [inline node/src/chain_spec.rs:206]:
```suggestion
	#[allow(deprecated)]
	ChainSpec::builder(
```

File: node/src/chain_spec.rs (hunk 1 above)
Comment (simonsso) [reply node/src/chain_spec.rs:167]: the point here is not us use the deprecated function

File: node/src/chain_spec.rs [hunk 3]
```diff
@@ -196,27 +164,21 @@ pub fn development_config(id: ParaId) -> ChainSpec {
...
 		Extensions {
 			relay_chain: "rococo-local".into(), // You MUST set this to the correct network!
 			para_id: id.into(),
 		},
-		WASM_BINARY.expect("WASM binary was not build, please build it!"),
 	)
+	.with_name("Parachain Eden Development")
+	.with_id("para_eden_dev")
+	.with_chain_type(ChainType::Development)
+	.with_genesis_config_patch(development_config_genesis(id))
```
Comment (simonsso) [inline node/src/chain_spec.rs:177]: Using the unused properties here

File: node/src/chain_spec.rs (hunk 2 above)
Comment (simonsso) [reply node/src/chain_spec.rs:208]: not needed

**Comment by simonsso** [comment]
- [x]  Checked with local built zombie net.

File: runtimes/eden/src/lib.rs [hunk 4]
```diff
@@ -213,6 +217,16 @@ pub type XcmFungibleBenchmarks = pallet_xcm_benchmarks::fungible::Pallet<Runtime
 type EventRecord =
 	frame_system::EventRecord<<Runtime as frame_system::Config>::RuntimeEvent, <Runtime as frame_system::Config>::Hash>;
 sp_api::impl_runtime_apis! {
+		impl sp_genesis_builder::GenesisBuilder<Block> for Runtime {
+		fn create_default_config() -> Vec<u8> {
```
Comment (aliXsed) [inline runtimes/eden/src/lib.rs:221]: Fix whitespace formatting

**Review by aliXsed — APPROVED** [review]
LGTM

**Pull request merged**